setup

```diff
@@ -155,6 +155,16 @@ install() {
 
     cmd_dnf_remove="dnf --installroot ${ROOTFS} remove ${PACKAGES_TO_REMOVE} -y"
     ${cmd_dnf_remove}
+    # check if "${ROOTFS}"/etc/yum.repos.d
+    if [ -z "$(find  "${ROOTFS}"/etc/yum.repos.d -mindepth 1 -maxdepth 1)" ]; then
```

Comment:
looks like we have an extra space after find.

Reply:
Ack